package.json

```diff
@@ -12,6 +12,7 @@
     "@webassemblyjs/wasm-edit": "1.9.0",
     "@webassemblyjs/wasm-parser": "1.9.0",
     "acorn": "^7.4.0",
+    "binary-search-bounds": "^2.0.4",
```

Comment:
Code is very small, I think we can put it in our utils

Reply:
> Code is very small, I think we can put it in our utils

@evilebottnawi Do you mean to copy this code verbatim and place it in webpack under `./lib/util/`? https://github.com/mikolalysenko/binary-search-bounds/blob/master/search-bounds.js

Reply:
Yep, maybe small rewriting too

Reply:
@evilebottnawi Here it is: https://github.com/webpack/webpack/pull/11316/commits/314987e7e08ee2788394ce3d8e892035a4a20a24